warehouse/templates/email/password-reset-unverified/subject.txt

```diff
@@ -0,0 +1,17 @@
+{#
+ # Licensed under the Apache License, Version 2.0 (the "License");
+ # you may not use this file except in compliance with the License.
+ # You may obtain a copy of the License at
+ #
+ # http://www.apache.org/licenses/LICENSE-2.0
+ #
+ # Unless required by applicable law or agreed to in writing, software
+ # distributed under the License is distributed on an "AS IS" BASIS,
+ # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ # See the License for the specific language governing permissions and
+ # limitations under the License.
+-#}
+
+{% extends "email/_base/subject.txt" %}
+
+{% block subject %}{% trans %}Password reset request{% endtrans %}{% endblock %}
```

Comment:
```suggestion
{% block subject %}{% trans %}Password reset request for unverified email address{% endtrans %}{% endblock %}
```

This should be different than the regular "password reset" email subject, otherwise they might get threaded together.